modules/server/src/main/scala/GrpcServer.scala

```diff
@@ -73,5 +77,51 @@ trait GrpcServer[F[_]] { self =>
 }
 
 object GrpcServer {
-  def apply[F[_]](implicit F: GrpcServer[F]): GrpcServer[F] = F
+
+  // def server[F[_]](S: GrpcServer[F])(implicit B: Bracket[F, Throwable]): F[Unit] =
+  //   B.bracket(S.start)(_ => S.awaitTermination)(_ => S.shutdown)
```

Comment:
Why is the alternative above commented?

Reply:
We are currently still on cats-effect 0.10.x. 
The commented version is possible after we have upgraded to 1.x